lib/task-data/tasks/install-ubuntu.js

```diff
@@ -17,7 +17,15 @@ module.exports = {
         completionUri: 'renasar-ansible.pub',
         version: 'trusty',
         repo: '{{api.server}}/ubuntu',
-        rootPassword: "RackHDRocks!"
+        netImage: '{{api.server}}/ubuntu/install/filesystem.squashfs',
+        rootPassword: "RackHDRocks!",
+        rootSshKey: null,
```

Comment:
All `null` and `[]` should be removed from options, because these value will violate the task-schema. 

If task-schema is enabled (there is `schemaRef` in the task definition), only those option that has valid default value can be put in the `options` list, if user want to check the full supported options, user should turn to lookup the schema.

Since you add a new option `interface`, you need to update the task-schema. ( this is the schema for ubuntu: https://github.com/RackHD/on-tasks/blob/master/lib/task-data/schemas/install-os-general.json,  you can copy centos to add a ubuntu specific option: https://github.com/RackHD/on-tasks/blob/master/lib/task-data/schemas/install-centos.json).